base64.js

```diff
@@ -37,7 +37,7 @@
   // [https://gist.github.com/1020396] by [https://github.com/atk]
   object.atob || (
   object.atob = function (input) {
-    var str = String(input).replace(/=+$/, '');
+    var str = String(input).replace(new RegExp("=+$"), ''); // #31: ExtendScript bad parse of /=
```

Comment:
@davidchambers `/[=]+$/` is probably ubiquitous since any reasonable regexp library will support character sets.  I don't know what engines support non-capture groups (which was @michaelficarra 's suggestion)

In the interest of parsimony `/[=]+$/` only adds 2 characters to the output while `/(?:)=+$/` adds 4 and `new RegExp` adds many more, so that is probably the best option